scripts/swarm64_run_tpc_benchmark_internal

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python3.6
```

Comment:
Nope, no good idea. Breaks on systems with Python 3.7/3.8. It needs a proper version check.